setup.cfg

```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length = 180
-ignore = E122,E123,E126,E127,E128,E731,E722,W504,W605
+ignore = E122,E123,E126,E127,E128,E203,E731,E722,W503,W504,W605
```

Comment:
Those ignore statements are a big worrying TBH, can we remove them?

Reply:
I would love to remove most of these (eventually) but I think that would be out of scope for this PR?

Reply:
As for the ignores I explicitly added (both are needed for black and flake8 to work together):
[E203](https://www.flake8rules.com/rules/E203.html): whitespace before ':'
[W503](https://www.flake8rules.com/rules/W503.html): line break occurred before a binary operator

Reply:
Hmm, I'm wondering, mostly to better understand the impact, if we were to remove most of the ignores, would we see an immediate increase in code changes here in the PR?

Reply:
No, but pre-commit CI's flake8 will probably blowup with complaints. Pre-commit CI by default runs pre-commit on all files, not just the changes, we can change this to be consistent with the local pre-commit behavior if we want.

these are changes that `black`/`darker` would address:
- [E122](https://www.flake8rules.com/rules/E122.html): under-indented continuation lines
- [E128](https://www.flake8rules.com/rules/E128.html): under-indented continuation lines
- [E126](https://www.flake8rules.com/rules/E126.html): over-indented continuation lines
- [E127](https://www.flake8rules.com/rules/E127.html): over-indented continuation lines
- [W503](https://www.flake8rules.com/rules/W503.html): line break occurred before a binary operator
- [W504](https://www.flake8rules.com/rules/W504.html): line break occurred after a binary operator
- [E123](https://www.flake8rules.com/rules/E123.html): align closing bracket with opening bracket's line
- [E203](https://www.flake8rules.com/rules/E203.html): whitespace before ':'

other changes we will need to manually fix:
- [E731](https://www.flake8rules.com/rules/E731.html): do not assign lambda, use def
- [E722](https://www.flake8rules.com/rules/E722.html): do not use bare except
- [W605](https://www.flake8rules.com/rules/W605.html): invalid escape sequence

Reply:
Oh interesting, I assumed the CI would also only just run against the changes, having consistency between dev and CI makes sense to me.

Reply:
Hm actually I'm not finding the option to change this (was very sure I'd seen it somewhere...). I believe pre-commit CI is effectively running `pre-commit run --all-files`. This has no impact on how `darker` works but means that `flake8` gets run on the entire code base. If we remove these undesired ignore statements flake8 will report an additional 700+ errors. Most will be relatively simple to address with the exception of E722 errors.